chat-plugins/musicbox.js

```diff
@@ -0,0 +1,163 @@
+					}
+					for (var i = 0; i < box.songs.length; i++) {
+						if (box.songs[i].title === target || ~box.songs[i].link === target) {
+							match = box.songs[i];
+							box.songs.splice(i--, 1);
+						}
+					}
+					if (!match) return this.sendReply('|html|The song "<b>' + target + '</b>" isn\'t there in your music box...');
+					fs.writeFileSync(FILE, JSON.stringify(musicboxes, null, 1));
+					return this.sendReply('|html|The song "<b>' + match.title + '</b>" has been successfully removed from your music box.');
+					break;
+					
+				case 'css':
+					var box = musicboxes[user.userid];
+					if (!target || !target.trim()) {
+						if (!this.canBroadcast()) return;
+						if (toId(box.css)) return this.sendReplyBox('Your music box css: <code>' + box.css + '</code>');
+						return this.sendReplyBox('You haven\'t set button css for your music box yet.');
+					}
+					if (toId(target) in {'remove':1, 'delete':1, 'none':1, 'hidden':1}) delete box.css;
+					else box.css = target.replace(/^["']/, '').replace(/["']$/, '');
+					fs.writeFileSync(FILE, JSON.stringify(musicboxes, null, 1));
+					this.parse('/musicbox');
+					return this.sendReply('Your music box\'s button CSS has been updated.');
+					break;
+
+				case 'removeall':
+					if (!musicboxes[user.userid].songs.length) return this.sendReply('You don\'t have any songs in your music box.');
+					if (!user.confirm) {
+						user.confirm = true;
+						return this.sendReply('WARNING: You are about to remove all of the songs in your music box. Use this command again if you\'re sure you want to do this.');
+					}
+					delete user.confirm;
+					musicboxes[user.userid].songs = [];
+					fs.writeFileSync(FILE, JSON.stringify(musicboxes, null, 1));
+					return this.sendReply('You have successfully removed all songs in your music box.');
+					break;
+
+				case 'delete':
+					if (!this.can('hotpatch')) return false;
+					var targetUser = Users.getExact(target) ? Users.getExact(target).name : target;
+					var box = musicboxes[toId(targetUser)];
+					if (!box) return this.sendReply(targetUser + ' doesn\'t have a music box...');
+
+					delete box;
+					fs.writeFileSync(FILE, JSON.stringify(musicboxes, null, 1));
+					return this.sendReply('You have successfully deleted ' + targetUser + '\'s music box.');
+			}
+		} else {
+			if (!this.canBroadcast()) return;
+			if (target.length > 18) return this.sendReply('The username "' + target + '" is too long.');
+			var targetUser;
+			if (!toId(target)) targetUser = user.name;
+			else targetUser = Users.getExact(target) ? Users.getExact(target).name : target;
+			var box = musicboxes[toId(targetUser)];
+			if (!box) return this.sendReplyBox(targetUser + ' doesn\'t have a music box...');
+			if (!box.songs.length) return this.sendReplyBox(targetUser + '\'s music box is empty...');
+
+			var total = [];
+			box.songs.forEach(function (song) {
+				console.log(song.css);
```

Comment:
If you're not using the music box management sub-commands (I guess?), the command'll just show you your (or someone else's) music box